{
  "Header": {
    "Original": "CVE-2021-25290 (An issue was discovered in Pillow before 8.1.1. In TiffDecode.c, there ...)",
    "Line": 173373,
    "ID": "CVE-2021-25290",
    "Description": "(An issue was discovered in Pillow before 8.1.1. In TiffDecode.c, there ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-2716-1}",
      "Line": 173374,
      "Type": "xref",
      "Bugs": [
        "DLA-2716-1"
      ]
    },
    {
      "Original": "- pillow 8.1.1-1",
      "Line": 173375,
      "Type": "package",
      "Package": "pillow",
      "Kind": "fixed",
      "Version": "8.1.1-1"
    },
    {
      "Original": "[buster] - pillow 5.4.1-2+deb10u3",
      "Line": 173376,
      "Type": "package",
      "Release": "buster",
      "Package": "pillow",
      "Kind": "fixed",
      "Version": "5.4.1-2+deb10u3"
    },
    {
      "Original": "NOTE: https://pillow.readthedocs.io/en/stable/releasenotes/8.1.1.html",
      "Line": 173377,
      "Type": "NOTE",
      "Description": "https://pillow.readthedocs.io/en/stable/releasenotes/8.1.1.html"
    },
    {
      "Original": "NOTE: https://github.com/python-pillow/Pillow/commit/e25be1e33dc526bfd1094bc778a54d8e29bf66c9",
      "Line": 173378,
      "Type": "NOTE",
      "Description": "https://github.com/python-pillow/Pillow/commit/e25be1e33dc526bfd1094bc778a54d8e29bf66c9"
    }
  ]
}